framework/encode/custom_vulkan_api_call_encoders.cpp

```diff
@@ -348,5 +348,12 @@ VKAPI_ATTR VkResult VKAPI_CALL CopyAccelerationStructureKHR(VkDevice
     return GetDeviceTable(device)->CopyAccelerationStructureKHR(device, deferredOperation, pInfo);
 }
 
+VKAPI_ATTR void VKAPI_CALL CmdCuLaunchKernelNVX(VkCommandBuffer commandBuffer, const VkCuLaunchInfoNVX* pLaunchInfo)
+{
+    // TODO
+    GFXRECON_LOG_ERROR("CmdCuLaunchKernelNVX encoding is not supported");
```

Comment:
Skip encoding CmdCuLaunchKernelNVX since pParams and pExtras  in VkCuLaunchInfoNVX couldn't be encoded correctly.